src/bun.js/bindings/ErrorCode.cpp

```diff
@@ -405,7 +405,7 @@ namespace ERR {
 
 JSC::EncodedJSValue INVALID_ARG_TYPE(JSC::ThrowScope& throwScope, JSC::JSGlobalObject* globalObject, const WTF::String& arg_name, const WTF::String& expected_type, JSC::JSValue val_actual_value)
 {
-    auto arg_kind = arg_name.startsWith("options."_s) ? "property"_s : "argument"_s;
+    auto arg_kind = arg_name.contains("."_s) ? "property"_s : "argument"_s;
```

Comment:
Use `'.'` instead of `""_s` since it's a single byte
```suggestion
    auto arg_kind = arg_name.contains('.') ? "property"_s : "argument"_s;
```